services/conservation_portfolios/src/routes/by_topic.ts

```diff
@@ -0,0 +1,30 @@
+import { Router } from 'restify-router';
+
+import { EHFunction } from '../util/types';
```

Comment:
In this case, can the routes be relative?

Reply:
they are relative

Reply:
This comment is because in another PR you said _"We don't use relative paths anymore"_

Reply:
in the frontend